logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserMapper.kt

```diff
@@ -209,6 +308,38 @@ internal class UserMapperImpl(
             defederated = false
         )
 
+    /**
+     * Null and default/hardcoded values will be replaced later when fetching known users.
```

Comment:
suggestion: add todo to it would be better in my opinion.

Reply:
removed this mapper, as it's not needed